tests failing from cosmos-sdk: `t.TmpDir` not defined

Not sure what is going on here.  They may be using a newer version of `go`, but I don't think that we updated the cosmos-sdk version lately?

```
vendor/github.com/cosmos/cosmos-sdk/testutil/network/network.go:172:34: t.TempDir undefined (type *testing.T has no field or method TempDir)
```

--- a/.github/workflows/tests.yaml
+++ b/.github/workflows/tests.yaml
@@ -29,8 +29,6 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           go-version: ${{ secrets.GOLANG_VERSION }}
-          kubernetes_service_host: localhost
-          kubernetes_service_port: 9999
       - run: make test-full
 
   coverage:

--- a/README.md
+++ b/README.md
@@ -87,7 +87,7 @@ To become a contributor, please see the guide on [contributing](CONTRIBUTING.md)
 
 ### Dependencies
 
-Akash is developed and tested with [golang 1.13.1+](https://golang.org/).  Building requires a working [golang](https://golang.org/) installation, a properly set `GOPATH`, and `$GOPATH/bin` present in `$PATH`.
+Akash is developed and tested with [golang 1.15.6+](https://golang.org/).  Building requires a working [golang](https://golang.org/) installation, a properly set `GOPATH`, and `$GOPATH/bin` present in `$PATH`.
 
  Most golang libraries will be installed via [`go modules`](https://github.com/golang/go/wiki/Modules),
  however the following packages:

